pkg/disk/utils.go

```diff
@@ -1336,7 +1336,11 @@ func checkRundVolumeExpand(req *csi.NodeExpandVolumeRequest) (bool, error) {
 	socketFile := filepath.Join(RundSocketDir, pvName)
 	if !utils.IsFileExisting(socketFile) {
 		klog.Infof("checkRundVolumeExpand: socketfile: %s not exists, fallback to runc expanding", socketFile)
```

Comment:
Update this log message?
```suggestion
		klog.Infof("checkRundVolumeExpand: socketfile: %s not exists, trying old path", socketFile)
```